sw/host/opentitanlib/src/transport/ti50emulator/emu.rs

```diff
@@ -31,7 +28,7 @@ use crate::transport::ti50emulator::Ti50Emulator;
 
 const TIMEOUT: Duration = Duration::from_millis(1000);
 const MAX_RETRY: usize = 5;
-const PATTERN: &[u8; 5] = b"READY";
+const PATTERN: &str = "CHIP READY";
```

Comment:
Is it always present in Emulator output ?

Reply:
Yes, the `ready()` method in `ports/host_emulation/src/chip.rs` first prints "CHIP READY" to standard output, and then immediately afterwards, provided that a control socket path was specified on the command line, it opens a connection to that socket and prints "READY" into that.